### PR TITLE
Add Run Length Encoding exercise

### PR DESCRIPTION
**Pull request by robtimp**
@masters3d

### CODE REVIEW

**Comment by masters3d** [comment]
travis probably rolled back to 10.11
could you change that back to 10.11 from 10.12 in the travis config?

**Comment by masters3d** [comment]
I think I fixed it.

File: exercises/run-length-encoding/RunLengthEncodingTest.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import XCTest
```
Comment (masters3d) [inline exercises/run-length-encoding/RunLengthEncodingTest.swift]: Can you add the swift 3 check here?

Comment (masters3d) [inline exercises/run-length-encoding/RunLengthEncodingTest.swift]: I think there are going to have a better flag later but for now I think having #if swift check would work.

**Comment by masters3d** [comment]
Thanks! we are getting close to release the swift3 version. I think as soon as xcode goes GM, we pull the trigger.

**Comment by robtimp** [comment]
Great!

**Pull request merged**